dev-util/cscope/cscope-15.9.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools elisp-common toolchain-funcs
+
+DESCRIPTION="Interactively examine a C program"
+HOMEPAGE="http://cscope.sourceforge.net/"
+SRC_URI="mirror://sourceforge/cscope/${P}.tar.gz"
+
+LICENSE="BSD GPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~ppc-aix ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x86-solaris"
+IUSE="emacs"
+
+RDEPEND=">=sys-libs/ncurses-5.2:0=
+	emacs? ( virtual/emacs )"
+DEPEND="${RDEPEND}
+	sys-devel/flex
+	virtual/pkgconfig
+	virtual/yacc"
+
+SITEFILE="50${PN}-gentoo.el"
+
+src_prepare() {
+	eapply_user
```

Comment:
`s/eapply_user/default/`. `default` eapplies the contents of `PATCHES=()` if there is one
(and there may be one later, so may as well save on future changes) and calls
`eapply_user` as well.

Reply:
This is entirely a question of the maintainer's style. cscope-15.8a-r2.ebuild has explicit eapply and eapply_user calls and I think that is perfectly fine.

However, while the Darwin patch has been taken care of upstream, cscope-15.7a-ocs-sysdir.patch still applies and shouldn't have been dropped.